WordPress/src/main/java/org/wordpress/android/ui/stats/StatsCommentsFragment.java

```diff
@@ -82,7 +82,7 @@ protected void updateUI() {
             return;
         }
 
-        if (mDatamodels[1] != null) { // check if comment-followers is already here
+        if (mDatamodels[1] != null && !isErrorResponse(1)) { // check if comment-followers is already here
```

Comment:
If it was safe to assume it was a CommentFollowersModel then this exception wouldn't have happened. I still think the safest thing to do here is explicitly check whether it's the expected type.